app/templates/items/detail/tabs/storage_content.hbs

```diff
@@ -1,7 +1,43 @@
 {{partial 'items/detail/tabs/_sub_tabs' }}
-
 <div class="row">
-  <div class="small-12 columns add-item" align="center">
-    <button class="button expand" disabled={{disableBoxPalletItemAddition}}>Add item to {{storageTypeName}}</button>
+  <div class="large-12 medium-9 small-9 columns add-item" align="center">
+    <button class="button expand" {{action 'openItemsSearch' item target=packageService}}
```

Comment:
The overlay is within the page, but the service has an `openItemsSearch` method. Calling it on any other page won't work. It should probably go to the controller.

Reply:
I'll do that change @patrixr .
Thanks. 
Please review API PR=> https://github.com/crossroads/api.goodcity/pull/891 as well.